src/CloudHsm/CloudHsm.Autorest/docs/Az.CloudHsmCmdlet.Design.md

```diff
@@ -0,0 +1,393 @@
+#### New-AzCloudHsm
```

Comment:
This file is used for design review. It is no need to be included in the module. Please remove it.